### PR TITLE
Review Auth section

### PR DESCRIPTION
**Pull request by hola-soy-milk**

### CODE REVIEW

File: content/4.auth/1.quickstart.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ curl \
 --url 'https://directus.example.com/items/posts'
 ```
 
-::callout{type="dev-docs" url="/auth/email-login"}
+::callout{type="dev-docs" url="/auth/email-login#refresh"}
```
Comment (phazonoverload) [inline content/4.auth/1.quickstart.md]: No inner page links please

File: content/4.auth/3.access-control.md [hunk 2]
```diff
@@ -78,20 +80,20 @@ A policy can toggle access to the App (Data Studio). If you only want a user to
...
+- **Archived**: a soft-deleted user.
 
 ## Limiting to Specific IP Addresses
 
 A policy can also have a specific allowlist of IP addresses, IP ranges, and CIDR blocks which allow access. Leave this empty to allow all IP addresses.
 
 ## Seats
 
-Seats in Directus are used in the context of licensing and billing for Directus Cloud. A user that occupies a seat meet at least one of the following criteria:
+Seats in Directus are used in the context of licensing and billing for Directus Cloud. A user that occupies a seat must meet at least one of the following criteria:
```
Comment (phazonoverload) [inline content/4.auth/3.access-control.md]: I don't think this is right. I think they "meet at least one" to be considered a seat.

File: content/4.auth/4.creating-users.md [hunk 3]
```diff
@@ -73,11 +73,13 @@ Inviting a user will send an email with an invite URL. Once invited, the user wi
...
 ```json [POST /users/invite]
 {
-	"email": "hello@example.com",
-	"role": "c86c2761-65d3-43c3-897f-6f74ad6a5bd7"
+  "email": "hello@example.com",
+  "role": "c86c2761-65d3-43c3-897f-6f74ad6a5bd7"
 }
 ```
 
+<!-- TODO: Fix broken link -->
```
Comment (phazonoverload) [inline content/4.auth/4.creating-users.md]: This isn't a todo here - the page is missing. Noted in Notion.

File: content/4.auth/1.quickstart.md (hunk 1 above)
Comment (hola-soy-milk) [reply content/4.auth/1.quickstart.md]: Consider them gone!

File: content/4.auth/3.access-control.md (hunk 2 above)
Comment (hola-soy-milk) [reply content/4.auth/3.access-control.md]: Noted, thank you!

Comment (hola-soy-milk) [reply content/4.auth/3.access-control.md]: I think there's a typo here then which I'll fix to "meets"

File: content/4.auth/4.creating-users.md (hunk 3 above)
Comment (hola-soy-milk) [reply content/4.auth/4.creating-users.md]: Got it! I wanted to make a note of it here so as to make sure we create the page, but doing so in Notion works as well!

**Comment by hola-soy-milk** [comment]
@phazonoverload Thanks for the feedback! It should be improved now. Could you please take a look?

File: content/4.auth/3.access-control.md (hunk 2 above)
Comment (phazonoverload) [reply content/4.auth/3.access-control.md]: I think strictly speaking both work but happy to adapt to this.

**Pull request merged**